catalog/ui/catalog-ui-search/src/main/webapp/js/HandlebarsHelpers.js

```diff
@@ -262,16 +262,19 @@ define([
                     return field;
                 }
             },
-          json: function (obj) {
-            return JSON.stringify(obj);
-          },
-          ifUrl: function(value, options){
-            if (value && value.toString().substring(0, 4) === 'http'){
-                return options.fn(this);
-            } else {
-                return options.inverse(this);
-            }
-          },
+            round: function(number, precision) {
+                return parseFloat(number).toPrecision(precision);
+            },
+            json: function (obj) {
+                return JSON.stringify(obj);
+            },
+            ifUrl: function(value, options){
+                if (value && value.toString().substring(0, 4) === 'http'){
```

Comment:
❗️ We shouldn't be using implicit booleans to check. Always explicit `value !== null` (or undefined, whichever is meant here)

Reply:
Changed to `value !== null`